can/map/map.js

```diff
@@ -523,7 +523,7 @@ var mapOverwrites = {	// ## can.Model#bind and can.Model#unbind
 	___set: function (base, connection) {
 		return function(prop, val){
 			base.apply(this, arguments);
-			if ( prop === connection.idProp && this._bindings ) {
```

Comment:
Looking at this here.  I wonder if it should be: `this.__bindEvents.lifecycleBindingsCount`.  Is `_` necessary when one is already in a `__`?